fe/fe-core/src/main/java/com/starrocks/qe/ResultReceiver.java

```diff
@@ -125,14 +127,15 @@ public RowBatch getNext(Status status) throws TException {
             LOG.warn("fetch result execution exception, finstId={}", finstId, e);
             if (e.getMessage().contains("time out")) {
                 // if timeout, we set error code to TIMEOUT, and it will not retry querying.
-                status.setStatus(new Status(TStatusCode.TIMEOUT, e.getMessage()));
+                status.setStatus(new Status(TStatusCode.TIMEOUT,
+                        String.format("Query exceeded time limit of %d seconds", expireSeconds)));
             } else {
                 status.setRpcStatus(e.getMessage());
                 SimpleScheduler.addToBlacklist(backendId);
             }
         } catch (TimeoutException e) {
             LOG.warn("fetch result timeout, finstId={}", finstId, e);
-            status.setStatus("query timeout");
+            status.setStatus(String.format("Query exceeded time limit of %d seconds", expireSeconds));
```

Comment:
You could get timeout value by  ConnectContext

Reply:
> You could get timeout value by ConnectContext

But ResultReceiver don't have such a member variable, I think it's not necessary to add ConnectContext to its member variable just for output error message

Reply:
what I mean you could use ConnectContext.get, not need  a new member variable.